flink-python/src/main/resources/META-INF/licenses/LICENSE.cloudpickle

```diff
@@ -1,32 +0,0 @@
-This module was extracted from the `cloud` package, developed by
```

Comment:
Why remove this license file?

Reply:
We do not bundle cloudpickle in jar anymore, so the license file in jar should be removed

Reply:
if it is no longer bundled please also remove the entry in the the root NOTICE and `licenses` directory.

Reply:
Thanks for your reminder Chesnay. But the `cloudpickle` still exists in the `build-target` of `flink-dist`, i.e. `flink/build-target/opt/python/cloudpickle-1.2.2-src.zip`. So I think there is no need to change the root NOTICE and `licenses` directory.

Reply:
@zentol As the `cloudpickle-1.2.2-src.zip` is still bundled to the binary release, its license file need to copy to the binary release. But `collect_license_files.sh` only collects the license files in jar, which means we need to add some code to ensure that the `LICENSE.cloudpickle` can be collected by the script. There are 2 solutions in my mind:
1. bundle the license file to `flink-python.jar`
2. let the `collect_license_files.sh` also collect the license files in the root `license` directory.

What do you think about those solutions?